el/index.js

```diff
@@ -82,16 +82,66 @@ module.exports = yeoman.Base.extend({
 
     var el;
     var pathToEl;
+    var pathToElements;
+    var pathToBowerComponents;
+    var pathToApp;
+    
+    // Let yoy spesify a path to the app folder
```

Comment:
There is a typo: `yoy spesify` should be `you specify`. Same mistake at lines 108 and 122.